sys-apps/systemd/systemd-250.3.ebuild

```diff
@@ -560,6 +560,8 @@ multilib_src_install_all() {
 	if use gnuefi; then
 		builddir_systemd_enable_service sysinit.target systemd-boot-update.service
 	fi
+	# Flatcar: enable systemd-sysext.service
+	builddir_systemd_enable_service sysinit.target systemd-sysext.service
```

Comment:
It's a nitpick, but could you put that below the `enable reboot.target` comment separated with an empty line? This section is basically following the contents of upstream's `90-systemd.preset`, so I would like to avoid mixing in our custom stuff like enabling systext here. Thanks!

Reply:
:+1: